src/components/Render_search_timeline.js

```diff
@@ -16,12 +16,15 @@ import {
 
 class Render_search_timeline extends React.Component {
   render() {
+    const { user_tl } = this.props;
+
+    const isUnavailable = get(user_tl, "unavailable", false);
     const data_user = get(this.props.user_tl, "graphql.user");
     const data_user_tl = get(
       this.props.user_tl,
       "graphql.user.edge_owner_to_timeline_media.edges"
     );
-    if (!data_user || !data_user_tl) {
+    if (!data_user || !data_user_tl || isUnavailable) {
```

Comment:
This is out of scope with this PR but this whole block is a copy/paste of the same in `src/components/Render_search_map.js`. It should be moved into a new re-usable component instead